### ISSUE TITLE
screen reader doesn't announce auto complete suggestion 

### ISSUE THREAD

**Issue by adil-s** [freeCodeCamp/freeCodeCamp#39135]
**Looking forward for reporting a security issue:**
Please report security issues by sending an email to `security@freecodecamp.org` instead of raising a GitHub issue.

**Describe the bug**
I've been learning programming from freecodecamp for a while. the autocomplete feature used to work well, but it seems that it has broken for a while. 
the issue is the screen reader (NVDA) doesn't announce the available suggestion and doesn't announce anything if suggestion is accepted using tab key. 
it used to work well. 
**To Reproduce**
1. enable screen reader and go to anyjava script  challange on freecodecamp 
2. focus the editor window and press (alt+f1 and then ctrl+e to enable accessibility)
3. type "console." use up/down arrow to access suggestion (NVDA should announce suggestion  but it doesn't)
4. press tab to accept the suggestion (NVDA should announce what suggestion has been accepted but it doesn't)
**Expected behavior**
I expected screen reader to announce the auto complete suggestion and I also expect it to let me know what suggestion has been accepted. 
**Screenshots**

**Desktop (please complete the following information):**
 - OS: windows 10 
 - Browser: chrome on windows, microsoft edge (chromium based on windows)
 - Version: [chrome: 83.0.4103.116 ], [microsoft edge:83.0.478.54 ] 

**Additional context**

**Comment by nhcarrigan** [comment]
To my knowledge, the autocomplete feature itself was turned off a while ago. This may be the reason your screen reader is not responding as it did previously.

**Comment by Ieahleen** [comment]
yes, it was turned off. It was decided that typing everything helps memorizing stuff better.

You will find that once you move on to an actual code editor, instead of the learning environment in Free Code Camp, you will have autocomplete back.

**Comment by adil-s** [comment]
it doesn't work. even when I trigger using ctrl+space.  I also mension in the issue that accepting suggestion doesn't give me any indication that code has been change.  I mean, screen reader (NVDA) should announce what suggestion has been accepted.

**Comment by adil-s** [comment]
when I press ctrl+space the suggestion list comes up, but I'm not able to explore the list using up/down arrow because NVDA keeps saying the lastword.

**Comment by Ieahleen** [comment]
the issue seems to be different then... it seems the autosuggestion has
been visually hidden, but has some functionality accessible by a screen
reader.

the autosuggestion feature has been turned off, or that was the intention,
so that students have to type everything to better remember it.

**Comment by ojeytonwilliams** [comment]
As @Ieahleen and @nhcarrigan said, it has been disabled. However, I think we should reinstate the functionality in accessibility mode if we can.

**Comment by nhcarrigan** [comment]
@ojeytonwilliams - It seems that there is some error with the autocomplete functionality. I was just working on a lesson, and typing `this.` followed by the `ENTER` key caused the editor to autocomplete to `this.collection`.

**Comment by ojeytonwilliams** [comment]
Yep, I see the problem, thanks @nhcarrigan.  It is invisibly (and inaudibly) switching between options when you use the arrow keys and then completing when you press tab or enter.  It is strange that the screen reader does not read out the word when you press tab.

**Comment by raisedadead** [comment]
Thanks for reporting.

IMHO, we should disable the autocompletion, but I can see it to be aid for a11y reasons. We should figure out how to get this working conditionally.

**Comment by moT01** [comment]
Is this still an issue @adil-s?

**Comment by raisedadead** [comment]
Hi, 

We are closing old issues/pull-requests that we think are stale. If you need still need help with this or are fixing this, please open a fresh issue/pull-request.

Thanks for your understanding. 

Happy contributing.
